### ISSUE TITLE
change Docs video events to "video" category

### ISSUE THREAD

**Issue by ethanlan14** [NuID/site#64]
Change the event tags for Google Analytics on videos from the Docs page to:

Category: video
Action: play
Label: docs_[title]

Currently videos on the Docs page have:

Category: docs_tile
Action: click
Label: [title]

The goal is to have all videos tracked under the "video" category, with the standard label format of [page]_[title].

**Comment by Nolan330** [comment]
https://github.com/nuid/site/commit/c42c6d3faeb8b8ef16c9ba34402aaa55b31ad75a

These are two separate events. We should avoid inference at the event granularity; when you infer at the event level, any model built on top is (should be) considered unusable. It seems inconspicuous, but this is how bias gets introduced into ML models and ✨ 🤖 `AI` ✨

I've kept the original event unchanged:
```
Category: docs_tile
Action: click
Label: [title]
```
Which fires when a user `click`s on the docs tile corresponding to `[title]`.

I've added a new event:
```
Category: video
Action: click
Label: docs_[title]
```
that fires when a user `click`s a video tile, in addition to the original event. It's a `click` action until we can intercept and forward an authentic `play` event (i.e. when the user clicks the play button in the embedded player), which seems to require unifying our videos under a paid Vimeo account.
